discovery-infra/tests/base_test.py

```diff
@@ -86,10 +86,26 @@ def generate_and_download_image(
             image_path=iso_download_path
         )
 
+    @staticmethod
+    def wait_until_hosts_are_registered(
```

Comment:
per the discussion in the channel, this should be unified with wait_until_host_are_discovered()

Reply:
in another issue. lots of the tests are depending on ```wait_until__host_are_discovered()`` and I think they mean a non-insufficient discovered host